hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx

```diff
@@ -137,7 +137,6 @@ cdef class AvellanedaMarketMakingStrategy(StrategyBase):
         self._start_time = start_time
         self._end_time = end_time
         self._min_spread = min_spread
-        self._latest_parameter_calculation_vol = s_decimal_zero
```

Comment:
If we remove this variable from the .pyx file we need to remove it also from the .pxd file